Sources/LiveViewNative/Modifiers/OpacityModifier.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OpacityModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 3/28/23.
+//
+
+import SwiftUI
+
+/// Sets the opacity for any element.
+///
+/// ```html
+/// <Text
+///     modifiers={
+///         opacity(@native, opacity: 0.5)
+///     }
+/// >Hello World!</Text>
+/// ```
+///
+/// ## Arguments
+/// * ``opacity``
+struct OpacityModifier: ViewModifier, Decodable, Equatable {
+    private let opacity: Double?
```

Comment:
This could be made non-optional, and it needs the `@_documentation(visibility:)` attribute.
```suggestion
    #if swift(>=5.8)
    @_documentation(visibility: public)
    #endif
    private let opacity: Double
```